### PR TITLE
(feat) O3-3357: Support editing person attributes in the Edit question modal

### PR DESCRIPTION
**Pull request by gitcliff**
## Requirements

- [x] This PR has a title that briefly describes the work done including the ticket number. If there is a ticket, make sure your PR title includes a [conventional commit](https://o3-docs.openmrs.org/docs/frontend-modules/contributing.en-US#contributing-guidelines) label. See existing PR titles for inspiration.
- [x] My work conforms to the [OpenMRS 3.0 Styleguide](https://om.rs/styleguide) and [design documentation](https://om.rs/o3ui).
- [ ] My work includes tests or is validated by existing tests.

## Summary

This PR adds support for editing person attribute fields using the Edit question modal in the interactive builder.

## Screenshots

https://www.loom.com/share/7262fa95a5f8403a87fd5254eca3df50

## Related Issue
https://openmrs.atlassian.net/browse/O3-3357

### CODE REVIEW

File: src/components/interactive-builder/edit-question-modal.component.tsx [hunk 1]
```diff
@@ -397,6 +422,40 @@ const EditQuestionModal: React.FC<EditQuestionModalProps> = ({
...
+                    initialSelectedItem={personAttributeTypes.find(
+                      (personAttributeType) =>
+                        personAttributeType?.uuid === questionToEdit.questionOptions?.attributeType,
+                    )}
+                  />
+                )}
+              </div>
+            )}
+
             {fieldType !== 'ui-select-extended' && questionToEdit.type !== 'patientIdentifier' && (
```
Comment (denniskigen) [inline src/components/interactive-builder/edit-question-modal.component.tsx:459]:
```suggestion
            {fieldType !== 'ui-select-extended' &&
              questionToEdit.type !== 'patientIdentifier' &&
              questionToEdit.type !== 'personAttribute' && (
```

Comment (denniskigen) [reply src/components/interactive-builder/edit-question-modal.component.tsx:459]: So we don't see the concept lookup field for `personAttribute` type questions.

**Review by denniskigen — CHANGES_REQUESTED** [review]

File: src/components/interactive-builder/edit-question-modal.component.tsx (hunk 1 above)
Comment (gitcliff) [reply src/components/interactive-builder/edit-question-modal.component.tsx:459]: oh yeah, thanks @denniskigen ..
had forgotten to push the commit for this

**Review by denniskigen — APPROVED** [review]
LGTM. Thanks, @gitcliff!

**Pull request merged**